docs/content/release_notes/supported_storage.md

```diff
@@ -1,18 +1,17 @@
 # Supported storage systems
 
-IBM® block storage CSI driver 1.8.0 supports different IBM storage systems as listed in the following table.
+IBM® block storage CSI driver 1.9.0 supports different IBM storage systems as listed in the following table.
 
 |Storage system|Microcode version|
 |--------------|-----------------|
-|IBM FlashSystem™ A9000|12.3.2.b or later|
-|IBM FlashSystem A9000R|12.3.2.b or later|
-|IBM Spectrum Virtualize™ family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)|7.8 and above, 8.x|
-|IBM Spectrum Virtualize as software only|7.8 and above, 8.x|
-|IBM DS8000® family|8.x and higher with same API interface|
+|FlashSystem™ A9000|12.3.2.b or later|
+|FlashSystem A9000R|12.3.2.b or later|
+|Spectrum Virtualize™ family including Spectrum Virtualize as software only, Spectrum Virtualize for Public Cloud, SAN Volume Controller (SVC) and FlashSystem® family members built with Spectrum® Virtualize (including FlashSystem 5xxx, 7xxx, 9xxx, 9xxx)|7.8 and above, 8.x|
```

Comment:
`Spectrum Virtualize as software only, Spectrum Virtualize for Public Cloud` has not been tested how do we want to declare we support it?

Reply:
1. Spectrum Virtualize as software support we've been claiming support for since the beginning.
I believe that it's the same testing as for the rest of the products (but I'm not sure).
The difference here, is just how I formatted the support content.
2. Spectrum Virtualize for Public Cloud (SV4PC) has been added per the email thread posted in the ticket.

Reply:
Any way both of them has not been tested at all.

Reply:
@erantzabari  Please let me know what you want listed here per the thread above. thanks

Reply:
Per Michelle statement - lets proceed with declaration of SV4PC, without any actual testing but due to the assumption of same API and same version.

Reply:
Thanks for clarify @erantzabari.